### ISSUE TITLE
Allow additional tags in comments `<b><p><li><ul>`

### ISSUE THREAD

**Issue by GregDomjan** [bsorrentino/maven-confluence-plugin#99]
Similar to Issue #69 Allow Pre tag, 

Related to "Project document generation" we would like to allow use of additional formatting tags within comment that are allowed by javadoc.

`<b><\b>` mapped to *  (ensure no leading trailing space)
`<br>` mapped to \\
`<p>` mapped to double carriage return

I haven't got any numbered lists, li might not be such an easy substitution?
`<ul> <li>` mapped to  *
`<ol> <li>` mapped to  #

It also seems that the existing formatting such as for `<pre>` applies to the parameters, but is not applied to the mojo description.

**Comment by bsorrentino** [comment]
Hi, thx for feedback

I'll keep in consideration add their support in milestone 4.8

**Comment by GregDomjan** [comment]
Thanks for the excellent plugin so far and your future time and consideration :)

**Comment by bsorrentino** [comment]
I've released the ```4.9-SNAPSHOT```

Please, test it and let me know if it's aligned with what you've asked

**Comment by GregDomjan** [comment]
Hi thanks for the inclusion, sorry for the slow response, this does seem aligned with what I've asked.

Gave it a run and got some change, however some parts didn't convert. 
The goal description/summary didn't have changes applied. 
The goal parameters are being formated, but had some limitation. 

Checked out the code change and I think 
 * the regex for bold has an issue with end tag.
 * the regex for list items is forcing requirement for </li>  but even when added isn't working for me.

Here as an example
```
	/**
	 * The Ant-like inclusion patterns used to select files to process.  
	 * The patterns must be relative to the directory given by the parameter {@link #workingDirectory}. 
	 * <p>
	 * <B>By default, the pattern list includes</B>
	 * <ul>
	 *  <li><code>&#42;&#42;/&#42;.?ar</code></li>
	 *  <li><code>&#42;&#42;/&#42;.dll</code>
	 * </ul>
	 */
```

**Comment by bsorrentino** [comment]
Hi 
thanks for feedbacks

# I've fixed

**Comment by bsorrentino** [comment]
I've released the ```4.9.1-SNAPSHOT``` with all the fix

Please, test it and let me know if it's aligned with what you've asked

**Comment by bsorrentino** [comment]
released as hotfix on ```4.9.1```
